unison-src/transcripts-round-trip/main.output.md

```diff
@@ -641,19 +641,34 @@ x = ()
       Put definitions in here that are expected to
       parse with a different hash after pretty-printing.
       """
+    
+    sloppyDocEval : Doc2
+    sloppyDocEval =
+      use Nat +
+      {{
+      Here's an example of an eval block that's technically a
+      lambda but should print as a backticked block (since old
+      docs in the wild still use this format).
+      
+      ```
+      1 + 1
+      ```
+      }}
```

Comment:
Here we see it using the backtick syntax even though the input doc had a lambda with a `_` parameter.